### DISCUSSION TITLE
Netlify doesn't allow us to deploy<title>

### DISCUSSION THREAD

**Discussion by bruno-carneiro-da-silva** [community/community#40146] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

<h1>Hello everyone!</h1> 
<p>It's been few days that I was trying to deploy a simple project through the Netlify web site and whatever the way I do the error<br> message has been appearing constantly. <br>
I'd like to understand whether is a bug or not. It's my first moment at the platform.<br>
Someone got the same issue?<p>

![Screen Shot 2022-11-26 at 08 22 44](https://user-images.githubusercontent.com/49591443/204086277-804ddd18-9b08-4dbe-9ad0-140a257c74c3.png)

**Comment by ossanna16** [comment]
Hi there @Brunosilva7 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by yoannchaudet** [comment]
Hello,

This does not sound like a GitHub Pages issue. I suggest opening a [support ticket](https://support.github.com/contact).

**Comment by bruno-carneiro-da-silva** [comment]
Gotcha sir,
But man whatever the browser I try to open up it gives the same problem. Is
it necessary to upgrade to a plan and then it will allows me to deploy?
All the best,

**Reply by chetjogia** [reply]
I'm having the same problem at the moment! Any help would be greatly appreciated

Having the same issue with configuring the app from railway as well, so looks like its a github/app problem?
